src/main.js

```diff
@@ -227,12 +228,30 @@ app.on('ready', function() {
 
     trayIcon.setToolTip(app.getName());
     trayIcon.on('click', function() {
+      if (process.platform === 'win32') {
+        if (config.minimizeToTray) {
+          if (mainWindow.isHidden) {
```

Comment:
It seems to work as 'toggle'. In my feel, clicking should always show the window, would you hear me your opinion?
(Slack always opens/focuses the window when clicking the tray icon.)